.github/workflows/unit_tests.yml

```diff
@@ -76,6 +76,7 @@ jobs:
       - name: Upgrade pip version
         run: |
           pip install --upgrade "pip>=21.3.1"
+          pip install pip-tools
```

Comment:
Why is this needed here? Can you pin the version?

Reply:
Removed.